.github/workflows/judoscale-solid_queue-test.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       matrix:
         gemfile:
           - Gemfile
+          - Gemfile-solid_queue-0-3
```

Comment:
New Gemfile to test against solid_queue v0.3 so we get coverage on minimum version supported